enhancements/observability/multicluster-observability-addon.md

```diff
@@ -0,0 +1,767 @@
+        'tls.key': "TLS key"
+        'ca-bundle.crt': "Certificate Authority certificate"
+    - apiVersion: opentelemetry.io/v1alpha1
+      kind: OpenTelemetryCollector
+      metadata:
+        name: spoke-otelcol
+        namespace: open-cluster-management
+      spec:
+        config: |
+          receivers:
+            jaeger:
+              protocols:
+                grpc:
+            otlp:
+              protocols:
+                grpc:
+                http:
+          processors:
+          exporters:
+            debug:
+            otlp:
+              tls:
+                insecure: true
+          service:
+            pipelines:
+              traces:
+                receivers: [jaeger,otlp]
+                processors: []
+                exporters: [otlp, debug]
+```
+
+
+#### Hypershift [optional]
+
+N/A
+
+### Drawbacks
+
+- MCOA configuration through the MultiClusterObservability: the MCO CR nowadays has an already extensive set of configuration fields, when designing the MCOA configuration, we will need to take extra caution as to not make this CR more complex and hard to navigate;
+- MCOA manifest sync: with MCOA being deployed by MCO we will need to set up a procedure to maintain the MCOA manifests that live in the MCO repo up to date.
+- CRD conflicts: MCOA will leverage the CRDs from other operators we will have to ensure that we will not be running into situations where two operators are managing the same CRD
+
+## Design Details
+
+### Open Questions [optional]
+
+The [addon-framework](ocm-addon-framework) is still lacking some features that will fully enable the behaviours described in this proposal namelly: 
+- [ ] Support for reconcile resources generated by the addon (Jira issues [LOG-5343](https://issues.redhat.com/browse/LOG-5343) and [TRACING-4148](https://issues.redhat.com/browse/TRACING-4148)). MCOA needs this otherwise it will not react on changes to those resources.
+- [ ] Support for multiple resources of the same kind in the ManagedClusterAddOn resource (Jira issues [LOG-5344](https://issues.redhat.com/browse/LOG-5344)). MCOA needs this otherwise it will not react on changes to those resources.
+
+### Test Plan
+
+TBD
+
+### Graduation Criteria
+
+TBD
+
+#### Dev Preview
+
+- Installation of the ClusterLogForwarder and OpenTelemetryCollector CRs.
+- Support for all the authentication methods supported by ClusterLogForwarder (i.e. static auth, mTLS auth, etc).
+- Support for Per-ClusterSet ClusterLogForwarder.
+- Health probing per Signal
+- Status reporting
+
+#### Dev Preview -> Tech Preview
+
+- Refined CLF/OTelCollector CRs usage
+
+#### Tech Preview -> GA
+
+TBD
+
+#### Removing a deprecated feature
+
+None
+
+### Upgrade / Downgrade Strategy
+
+None
+
+### Version Skew Strategy
+
+None
+
+### Operational Aspects of API Extensions
+
+TBD
+
+#### Failure Modes
+
+- Cert Manager not installed: this will not allow the addon to create it's own CA and provision mTLS certificates
```

Comment:
Maybe it is an antipattern but, since we have access from the addon to the Hub cluster... would it be crazy to install the `cert-manager` from the addon if not present?